src/gui/folder.cpp

```diff
@@ -180,12 +180,22 @@ QString Folder::alias() const
 QString Folder::path() const
 {
     QString p(_path);
-    if( ! p.endsWith(QLatin1Char('/')) ) {
-        p.append(QLatin1Char('/'));
+    if( ! p.endsWith(QDir::separator()) ) {
+        p.append(QDir::separator());
     }
```

Comment:
I think we can not accept this change. It is important that we always use '/' as path delimiter. Only if the path is displayed to the user, we convert it to the platform specific delimiters.

Can you rewrite this patch without this change please?